src/app.py

```diff
@@ -259,8 +259,11 @@ async def add(r: AddObjectRequest, background_tasks: BackgroundTasks):
 async def tick():
     logger.debug(f"Starting to tick scene: {state.tick_number}")
     actions = await fetch_actions()
-    actions.extend([Entity.deserialize(pending_action) for pending_action in state.scene.pending_actions])
+    logger.debug(f"Returned fetched actions")
```

Comment:
Remember to divide logically unique parts of code with empty lines during the rebase